### PR TITLE
HOTFIX React hook call bug, multiple react versions

### PR DESCRIPTION
**Pull request by hugobarragon**
Fix react bug, show stopper when using this package

### CODE REVIEW

**Comment by hugobarragon** [comment]
Fixes bug https://github.com/knapcio/react-leaflet-pixi-overlay/issues/5  

As react says, there can only be 1 version of react, as this is a react library it cant have a direct dependecy of react, if so, when user instals this lib it will install react again, react is a dev-dep
https://reactjs.org/warnings/invalid-hook-call-warning.html

image

**Comment by knapcio** [comment]
Awesome! Thank you for your commitment! I will release a npm package soon :)

**Comment by hugobarragon** [comment]
> Awesome! Thank you for your commitment! I will release a npm package soon :)

no problem, if you need future help here you can ask me, will it be realeased today?

**Comment by knapcio** [comment]
Fantastic, thank you. Just released at the moment :)

File: package.json [hunk 1]
```diff
@@ -17,11 +17,18 @@
...
+  "peerDependencies": {
+    "react": ">=16.8.0",
+    "leaflet": ">=1.2.0",
+    "react-leaflet": ">=2"
+  },
+  "devDependencies": {
     "react": "^16.12.0",
     "react-leaflet": "^2.6.1",
-    "use-leaflet": "^1.6.1"
+    "leaflet": "^1.6.0",
```
Comment (knapcio) [inline package.json:32]: I had to remove this comma as it wasn't a valid JSON :)

Comment (hugobarragon) [reply package.json:32]: yes no problem thanks for noticing it, that mistake escaped me :)

**Pull request merged**